plugins/tech-radar/src/components/RadarTimeline/RadarTimeline.tsx

```diff
@@ -80,9 +80,7 @@ const RadarTimeline = (props: Props): JSX.Element => {
                   {timeEntry.ring.name ? timeEntry.ring.name : ''}
                 </TableCell>
                 <TableCell align="left">
-                  {timeEntry.date.toLocaleDateString()
-                    ? timeEntry.date.toLocaleDateString()
-                    : ''}
+                  {timeEntry.date ? timeEntry.date : ''}
```

Comment:
Note that this field SHOULD not be a string. The types say that it has to be a Date. Most users will be supplying it as a Date, and their experience will be pretty broken.

![Screenshot 2023-06-20 at 16 11 52](https://github.com/backstage/backstage/assets/3097461/3abb9ae8-13f7-4ba2-abce-0383e48c3266)

I understand that it would be convenient to make this data field support both string and Date in order to make it possible to load from JSON etc. But if we do that, then the type definition should be changed to `string | Date`, and this part of the code needs to parse the string and format it correctly. And it should do so in accordance [with this ADR](https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr012-use-luxon-locale-and-date-presets.md) preferably :)

Reply:
Thanks for explanation. I understood what my issue was now and that the example code actually requires to convert the string to date. https://github.com/backstage/backstage/commit/e10cffa02e05b0bf262575e51950d0c2881dbf30#diff-e56a868ce6d8c425a4b183d7c2c5cc511259e79245d2351bb3cdc1d80f9256acR83

I just thought it was an example didn't realize this. Thanks for your feedback! I'll close the PR.